tensorflow/core/kernels/sparse_tensor_dense_matmul_op_gpu.cu.cc

```diff
@@ -1,11 +1,8 @@
-/* Copyright 2015 The TensorFlow Authors. All Rights Reserved.
-
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
-
     http://www.apache.org/licenses/LICENSE-2.0
-
```

Comment:
I don't think these three blank lines should be removed from the copyright header. They're part of the standard copyright header format/structure.